types/lodash/lodash-tests.ts

```diff
@@ -5898,6 +5898,14 @@ fp.now(); // $ExpectType number
     _.values(numDict); // $ExpectType AbcObject[]
     _.values(list); // $ExpectType AbcObject[]
     _.values(abcObject); // $ExpectType (string | number | boolean)[]
+    const abcObjectLiteral: {
+        a: 'a';
+        b: 'b';
+        c: 'c';
+    } = anything;
+    _.values(abcObjectLiteral); // $ExpectType ('a' | 'b' | 'c')[]
+    enum AbcEnum { a = 'a', b = 'b', c = 'c' };
+    _.values(AbcEnum); // $ExpectType AbcEnum[]
```

Comment:
Interesting… is the current `string[]` behaviour intended? I still would have thought the second overload would have been used.

I guess I'll have to go back to listing my enum values in an array manually!

Reply:
> But I don't know if that's possible in typescript

Wouldn't that be `Array<T[keyof T] | keyof T>`?

Reply:
> TS will compile it with reverse mappings as well

Actually, I just realised this is not the case.

https://www.typescriptlang.org/play/index.html#src=enum%20Baz%20%7B%0D%0A%20%20%20%20A%20%3D%20'a'%2C%0D%0A%20%20%20%20B%20%3D%20'b'%0D%0A%7D

``` ts
enum Baz {
    A = 'a',
    B = 'b'
}
```

generates

``` ts
var Baz;
(function (Baz) {
    Baz["A"] = "a";
    Baz["B"] = "b";
})(Baz || (Baz = {}));
```

The syntax you posted a moment ago was using `:` instead of `=`. AFAIK this is invalid syntax.

Reply:
Hi @aj-r, any ideas?